reactor-core/src/main/java/reactor/core/publisher/StateLogger.java

```diff
@@ -49,7 +49,7 @@ void log(String instance,
 					formatState(committedState, 64)), new RuntimeException());
 		}
 		else {
-			this.logger.trace(String.format("[%s][%s][%s][%s-%s]",
+			this.logger.trace(String.format("[%s][%s][%s][\n\t%s\n\t%s]",
```

Comment:
With this change it's much easier to compare the state changes as they are stacked on top of each other so the differences are immediately visible as opposed to a horizontal presentation.